app/ts/components/pages/WebsiteAccess.tsx

```diff
@@ -412,15 +388,14 @@ const AdvancedSettings = ({ websiteAccess }: { websiteAccess: Signal<WebsiteAcce
 	)
 }
 
-const BlockRequestSetting = ({ websiteAccess }: { websiteAccess: Signal<WebsiteAccess | undefined> }) => {
+const BlockRequestSetting = ({ websiteAccess }: { websiteAccess: Signal<WebsiteAccess> }) => {
 	const setWebsiteExternalRequestBlocking = async (shouldBlock: boolean) => {
 		if (!websiteAccess.value) return
 		sendPopupMessageToBackgroundPage({ method: 'popup_blockOrAllowExternalRequests', data: { website: websiteAccess.value.website, shouldBlock } })
 	}
 
-	if (!websiteAccess.value) return <></>
-
-	const requestBlockMode = useComputed(() => websiteAccess.value?.declarativeNetRequestBlockMode)
+	const requestBlockMode = useComputed(() => websiteAccess.value.declarativeNetRequestBlockMode)
+	const website = useComputed(() => websiteAccess.value.website)
```

Comment:
Should this be `useOptionalComputed`?  There are a few other places in this PR where you introduced `useComputed` but maybe you meant to do `useOptionalComputed`.